tests/src/core/physics/SpringJoint.test.ts

```
@@ -0,0 +1,193 @@
+  beforeEach(function () {
+    rootEntity.clearChildren();
+    engine.sceneManager.activeScene.physics.gravity = new Vector3(0, -9.81, 0);
+  });
+
+  it("minDistance", function () {
+    engine.sceneManager.activeScene.physics.gravity = new Vector3(0, 0, 0);
+    const boxEntity = addBox(new Vector3(1, 1, 1), DynamicCollider, new Vector3(0, 5, 0));
+    const boxEntity2 = addBox(new Vector3(1, 1, 1), DynamicCollider, new Vector3(2, 5, 0));
+    const collider2 = boxEntity2.getComponent(DynamicCollider);
+    const joint = boxEntity2.addComponent(SpringJoint);
+    const collider = (joint.connectedCollider = boxEntity.getComponent(DynamicCollider));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)